content/en/docs/gke/deploy/project-setup.md

```diff
@@ -76,6 +76,10 @@ Follow these steps to set up your GCP project:
 
     Refer to [Anthos Service Mesh documentation](https://cloud.google.com/service-mesh/docs/archive/1.4/docs/gke-install-new-cluster#setting_credentials_and_permissions) for details.
 
+    For more information, see the following issues:
+
+    * [kubeflow/website #2121](https://github.com/kubeflow/website/issues/2121) Projects which have never had a k8s cluster before might need to create and delete a temp cluster to activate this.
```

Comment:
@theofpa Thank you again. What do you think of this? Rephrased with more context:

```suggestion
    If you encounter a `Workload Identity Pool does not exist` error, refer to the following issue:

    * [kubeflow/website #2121](https://github.com/kubeflow/website/issues/2121)
    describes that creating and then removing a temporary Kubernetes cluster may
    be needed for projects that haven't had a cluster set up beforehand.
```

Reply:
Much better, thanks @8bitmp3

Reply:
No problem @theofpa If you liked that revision, please implement it and we'll LGTM and approve it asap.